src/manager/CacheManager.py

```diff
@@ -1,6 +1,6 @@
 import logging
 import logging.config
-logging.config.fileConfig("logging.conf")
+logging.config.fileConfig('logging.conf')
```

Comment:
let's try to use the same way to do stuff if ' or " in the import file should be the same in all the files not a mix